content/10.diagnostics.md

```diff
@@ -81,11 +91,18 @@ The authors are now working to get the assay into clinical use [@url:https://www
 
 Importantly, false-positives can occur due to the cross-reactivity with other antibodies according to the clinical condition of the patient [@url:https://www.fda.gov/media/136625/download].
 Therefore, this test should be used in combination with RNA detection tests [@url:https://www.fda.gov/media/136625/download].
+Serological tests also require waiting until a patient has had an imune system reaction sufficient to detect, which means that these tests are of limited use for diagnosis in the early stages of an infection.
+The limitation on timing makes serological tests vfar less useful for enabling test-and-trace strategies.
 
 ### Possible Alternatives to Current Practices for Identifying Active Cases
 
-- Are there other approaches that have worked for diagnosing other viruses at a rapid pace in large numbers of people?
-- What are some approaches people are currently testing for detecting live viruses, especially SARS-CoV-2?
+Clinical symptoms are similar enough to other types of pneumonia that purely symptomatic diagnosis is unlikely to be sufficient.
+However, related approaches for diagnosis of cases exist, though most of these are only likely to identify serious and symptomatic cases.
+
+X-ray diagnostics have high sensitivity but low specificity in some studies. [@doi:10.1148/radiol.2020200642]
+Others have shown that some doctors seems to be able to better distinguish, though sensitivity in this case was lower [@doi:10.1148/radiol.2020200823].
```

Comment:
Others have shown that some doctors seem to be able to distinguish better, even though sensitivity, in this case, was lower

Reply:
Maybe something like "Other studies have shown that specificity varies between radiologists, though the sensitivity reported here was lower"
(Assuming that's consistent with what the study says, I didn't check it!)